src/sql/workbench/contrib/notebook/browser/models/modelInterfaces.ts

```diff
@@ -400,6 +402,8 @@ export interface INotebookModel {
 
 	getApplicableConnectionProviderIds(kernelName: string): string[];
 
+	updateActiveCell(cell: ICellModel): void;
```

Comment:
Why not just have the property and then define the setter with the logic for this? Not sure why you'd need a separate method